lib/lago/__init__.py

```diff
@@ -67,16 +67,16 @@ def _ip_in_subnet(subnet, ip):
         only 255.255.255.0 masks allowed
 
     Args:
-        subnet (str): Strign containing the three first elements of the decimal
-            representation of a subnet (X.Y.Z) or a full ip (X.Y.Z.A)
-        ip (str or int): Decimal ip representation
+        subnet (str): String containing the three first elements of the decimal
+            representation of a subnet (X.Y.Z) or a full IP (X.Y.Z.A)
+        ip (str or int): Decimal IP representation
 
     Returns:
-        bool: ``True`` if ip is in subnet, ``False`` otherwise
+        bool: ``True`` if IP is in subnet, ``False`` otherwise
     """
     return (
-        _create_ip(subnet, 1) == _create_ip(ip, 1)
-        or '0.0.0.1' == _create_ip(ip, 1)
+        _create_ip(subnet, 1) == _create_ip(ip, 1) or
+        '0.0.0.1' == _create_ip(ip, 1)
```

Comment:
We are not using pycharms, there's a pull request on how to set up the dev environment, will try to push it